### ISSUE TITLE
Timeline: error when opening image popup for the first time

### ISSUE THREAD

**Issue by AlexTr** [unacms/UNA#1342]
It can be reproduced on una.io under logged in user.

JS error:

```
Uncaught TypeError: jQuery(...).on(...).froalaEditor is not a function
```

or

```
 TypeError: jQuery(...).tinymce is not a function
```

**Comment by AntonLV** [comment]
The issue appears when the "Post to Timeline" block is hidden.

**Comment by AlexTr** [comment]
fixed in #1345
